package.json

```diff
@@ -27,47 +27,47 @@
     "test:watch": "npm run test -- --watchAll"
   },
   "dependencies": {
-    "@storybook/theming": "^5.0.6",
-    "async-validator": "^1.11.0",
-    "body-scroll-lock": "^2.6.1",
-    "copy-to-clipboard": "^3.1.0",
-    "i18next": "^15.0.9",
-    "i18next-browser-languagedetector": "^2.2.3",
+    "@storybook/theming": "^5.1.9",
```

Comment:
is this the right place for moving this dependency to the dev dependencies?

Reply:
yeah sure I can do that